mne/__init__.py

```diff
@@ -18,6 +18,45 @@
 
 __version__ = '0.18.dev0'
 
+# Set the default backend
```

Comment:
I would move all this into some existing utils. ie `.utils` `.viz.utils` and import it.

If does not make sense in any of the existing utils, I would create something new. If you want to keep it separate it while developing, I would put in a new file somewhere else while we figure out where shall it go.

Reply:
Thanks for the feedback! I like the idea. I'm gonna work on it.